fabfile.py

```diff
@@ -356,6 +365,21 @@ def update(c, wiredtiger_branch=None, testy_branch=None):
         raise Exit("One or more errors occurred during update. Please retry the " \
                    f"update or run 'fab start' to restart {testy}.")
 
+# Execute the validate function defined in the workload.
+@task
+def validate(c):
```

Comment:
@ruby-oujo, do you think it would be more flexible if this would take a workload? Or is it ok to rely on the currently defined one?

Reply:
You can only validate the workload that has just run right? Since you need the database files. Is there a way to validate a different workload?

Reply:
Yes I agree. If a workload has a very general way to verify db files, like "sample", it could be applicable to a database that was generated with a different workload.